package.json

```
@@ -4,45 +4,31 @@
     "packages/*"
   ],
   "options": {
-    "mocha": "--full-trace --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js",
+    "mocha": "--full-trace --require @babel/register --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js",
     "mocha_tdd": "--full-trace --watch --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js"
   },
   "scripts": {
-    "test": "npm run lint && npm run pretty-check && npm run testonly && npm run check",
-    "testonly": "lerna run build && babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",
-    "t": "npm run testonly",
-    "lint": "eslint src packages/**/src || (printf '\\033[33mTry: \\033[7m npm run lint -- --fix \\033[0m\\n' && exit 1)",
+    "build": "lerna run build",
+    "test": "yarn run lint && yarn run pretty-check && yarn run testonly && yarn run check",
+"testonly": "lerna run build && babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",    "t": "yarn run testonly",
+    "lint": "eslint src packages/**/src || (printf '\\033[33mTry: \\033[7m yarn run lint -- --fix \\033[0m\\n' && exit 1)",
```

```suggestion
    "lint": "eslint src 'packages/**/src' || (printf '\\033[33mTry: \\033[7m yarn run lint -- --fix \\033[0m\\n' && exit 1)",
```

Never trust shell globbing to be consistent. Use ESLint's globbing (please check this works!)

---

This is the script that is already in place:
https://github.com/graphql/graphql-language-service/blob/master/package.json#L14

perhaps we should open a different PR to fix this then?

---

On closer inspection, I think that should be `packages/*/src`. Anyway, it's fine because there's never a `packages/src` folder so we don't have to worry 👍